src/AbsoluteCellRange.ts

```diff
@@ -55,6 +55,21 @@ export class AbsoluteCellRange implements SimpleCellRange {
     return this.start.sheet
   }
 
+  public static fromSimpleCellAddresses(start: SimpleCellAddress, end: SimpleCellAddress): AbsoluteCellRange {
+    const width = end.col - start.col
+    const height = end.row - start.row
+
+    if (Number.isFinite(height) && Number.isFinite(width)) {
+      return new AbsoluteCellRange(start, end)
+    }
+
+    if (Number.isFinite(height)) {
+      return new AbsoluteRowRange(start.sheet, start.row, end.row)
+    }
+
+    return new AbsoluteColumnRange(start.sheet, start.col, end.col)
```

Comment:
Well, in theory, it will be handled by this factory method. It added a unit test for that as suggested in another comment. But keep in mind, that such a range will never be created because the formula syntax does not allow it. There are only cell ranges (`A1:D4`), column ranges (`A:D`) and row ranges (`1:4`).